.github/workflows/reusable-performance.yml

```
@@ -15,7 +15,7 @@ on:
         description: 'The version being used for baseline measurements.'
         required: false
         type: 'string'
-        default: '6.1.1'
+        default: '6.7'
```

Should we set this to `6.7.1.` instead?
```suggestion
        default: '6.7.1'
```

---

I thought about that, but there really isn't a need to update this to the absolute latest version. If the change to 6.7 doesn't cause any unforeseen side effects to our data collection, I think it makes sense to start bumping this on each major release going forward.

---

I think it should be fine to use `6.7` and not `6.7.1`. After all, we're comparing performance between major releases, not minor ones. If we start with 6.7.1, it's possible (though unlikely) a performance positive change in 6.7.1 would not be included in the next report.